docs/src/extend/custom-rules.md

```diff
@@ -684,6 +635,57 @@ Finally, comments can be accessed through many of `sourceCode`'s methods using t
 
 Shebangs are represented by tokens of type `"Shebang"`. They are treated as comments and can be accessed by the methods outlined above.
 
+### Accessing Variable Scopes
+
+**Deprecated:** Use `SourceCode#getScope()` instead of `context.getScope()`.
+
+The `SourceCode#getScope(node)` method returns the scope of the given node. It is a useful method for finding information about the variables in a given scope and how they are used in other scopes.
```

Comment:
This section starts with bolded **Deprecated** so it looks like the whole Accessing Variable Scopes is deprecated. Perhaps this could be just `**Note:**`?

Reply:
We generally use this format for pointing out when things are deprecated. Maybe just moving it below the first paragraph would help?

Reply:
Looks better now, thanks!